lib/src/simulations/simulations.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:simulate/src/custom_items/simulation_card.dart';
+import 'package:simulate/src/simulations/toothpick.dart';
+import 'package:simulate/src/simulations/langton_ant.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+// import 'package:async/async.dart';
+
+class Simulations with ChangeNotifier {
+  static var _favorites = [-1, -1];
+  final _algorithm = [0, 1];
```

Comment:
I think a better way would be to store an `int` value such that all widgets having `ID` up to that number would be shown. However, alongside, we'd make a `list` of the excluded elements. This will optimize memory, as mostly we'd only need to hide certain simulations while making them, and so the exclusion list would remain particularly small. As for this list, when the number of simulations increases, the list would very large and occupy a lot of memory.